zerver/tests/test_bots.py

```diff
@@ -907,3 +907,27 @@ def test_patch_bogus_bot(self):
         result = self.client_patch("/json/bots/nonexistent-bot@zulip.com", bot_info)
         self.assert_json_error(result, 'No such user')
         self.assert_num_bots_equal(1)
+
+    def test_create_outgoing_webhook_bot(self, **extras):
+        # type: (**Any) -> None
+        self.login(self.example_email('hamlet'))
+        bot_info = {
+            'full_name': 'Outgoing Webhook test bot',
+            'short_name': 'outgoingservicebot',
```

Comment:
Sorry I missed this comment.  Good point!  It might still be interesting to test that duplicate bot user creation fails, but I don't think it's important enough to block this PR from being merged.